clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -459,6 +463,31 @@ private void handleColumnCommand() {
     }
   }
 
+  /**
+   * Handles the command execution for Objects based on command type and the command line options.
+   */
+  private void handleOwnerCommand() {
+    String url = getUrl();
+    FullName name = new FullName(line);
+    String metalake = name.getMetalakeName();
+    String entityName = line.getOptionValue(GravitinoOptions.NAME);
+
+    if (CommandActions.DETAILS.equals(command)) {
+      new OwnerDetails(url, ignore, metalake, entityName, entity).handle();
+    } else if (CommandActions.UPDATE.equals(command)) {
+      String owner = line.getOptionValue(GravitinoOptions.USER);
+      String group = line.getOptionValue(GravitinoOptions.GROUP);
+
+      if (owner != null && group == null) {
+        new SetOwner(url, ignore, metalake, entityName, entity, owner, false).handle();
+      } else if (owner == null && group != null) {
+        new SetOwner(url, ignore, metalake, entityName, entity, group, true).handle();
+      } else {
+        System.err.println(ErrorMessages.INVALID_OWNER_COMMAND);
+      }
+    }
```

Comment:
if the command is not "details" nor "update", it does nothing, it would be better to output something, let the user know the command is not supported?

Reply:
Done.